.vib/nginx/goss/goss.yaml

```diff
@@ -0,0 +1,32 @@
+file:
+  /opt/bitnami/nginx/conf/nginx.conf:
+    exists: true
+    filetype: file
+    mode: "0664"
+    owner: root
+    group: root
+command:
+  user-id-test:
+    exec: if [ "$(id -u)" -eq 0 ]; then exit 1; fi
+    exit-status: 0
+    stdout: []
+    stderr: [] 
+  nginx-config-files:
+    exec: nginx -t 
+    exit-status: 0
+    stdout: []
+    stderr: 
+      - "nginx.conf syntax is ok"
+      - "nginx.conf test is successful"  
+  nginx-reload:
+    exec: nginx -s reload
+    exit-status: 0
+    stdout: []
+    stderr: 
+      - "signal process started"
+  nginx-home:
+    exec: cat /opt/bitnami/nginx/html/index.html
+    exit-status: 0
+    stdout: 
+      - "If you see this page, the nginx web server is successfully installed"
+    stderr: 
```

Comment:
I think it might be better to query the server with `curl` and then compare the result with the file in disk. That way, you are double-checking that the server is operational and it returns the expected file.

Another approach would be to move this to the `file` section (I think it makes more sense) & implement a simple `curl` test here, which IMHO is more valuable than checking the static index file.

Reply:
Indeed, this check is now applied as well.